kong/db/declarative/init.lua

```diff
@@ -755,9 +756,20 @@ end
 
 
 function declarative.load_into_cache_with_events(entities, meta, hash)
+  local ok, err = ngx.shared.kong:add("declarative:flips", 0, 60)
```

Comment:
Better make the magic number `60` a module variable and export it, so that https://github.com/Kong/kong/pull/6121/files#diff-35cfffb01c3d2bebf06bab3805ed22c1R101 and https://github.com/Kong/kong/pull/6121/files#diff-35cfffb01c3d2bebf06bab3805ed22c1R101 can use it as well.